src/rebar_prv_update.erl

```diff
@@ -168,9 +168,9 @@ update_deps_list(Pkg, PkgVsn, Deps, HexRegistry, State) ->
                                 cmpl(Dep1, rm_ws(Vsn), HexRegistry, State,
                                      DepsListAcc, fun ec_semver:lt/2);
                             {_, <<"==", Vsn/binary>>} ->
-                                [{Dep, Vsn} | DepsListAcc];
+                                [{AppName, {pkg, Dep, Vsn, undefined}} | DepsListAcc];
```

Comment:
is the `undefined` here (and elsewhere) meant to eventually hold a hash (as in the lockfile)? do we need to do this?